### ISSUE TITLE
Migrate away from gs://kops-ci

### ISSUE THREAD

**Issue by ameukam** [kubernetes/k8s.io#2625]
Part of umbrella issue to migrate the kubernetes project away from use of GCP project google-containers: #1469

- [x] Create a new GCS bucket writable by k8s-infra-prow-build GKE cluster (`kops-ci` to `k8s-infra-kops-ci-results`)
- [x] Give kOps maintainers admin access to this bucket
- [x] Add a canary jobs (duplicate of the ones writing to gs://kops-ci) that pushes to the new bucket
	- ensure it's building and pushing appropriately
- [ ] update jobs that pull from the new bucket
- [x] get rid of the jobs that runs on the "default" cluster	

/sig cluster-lifecycle
/area jobs
/help wanted

/assign @justinsb @johngmyers 
For Kops

/assign @spiffxp 
For wg-k8s-infra

**Comment by spiffxp** [comment]
/unassign
I'm not actively working this, though I am happily reviewing @ameukam's PRs

**Comment by spiffxp** [comment]
/milestone clear
Clearing from milestone because migrating all of the kops jobs over is likely to affect our spend in a non-significant way

**Comment by upodroid** [comment]
This migration is complete.

Are we missing something @ameukam ?

**Comment by ameukam** [comment]
I see references of the bucket: https://cs.k8s.io/?q=https%3A%2F%2Fstorage.googleapis.com%2Fkops-ci&i=nope&files=&excludeFiles=&repos=. 🤔

**Comment by justinsb** [comment]
I think we're really close, we were able to set up the 1.30 jobs with no kops-ci at all.  I think there are still a few 1.28 / 1.29 jobs that point to kops-ci, but we can now repoint them following the example of 1.30 (we were waiting on releases on 1.28 / 1.29, I think they're now ~done)

**Comment by ameukam** [comment]
/close

What remaining is part of some legacy stuff and will probably go away.
